packages/next/server/post-process.ts

```diff
@@ -169,7 +169,7 @@ class FontOptimizerMiddleware implements PostProcessMiddleware {
 
     let preconnectTag = ''
     preconnectUrls.forEach((url) => {
-      preconnectTag += `<link rel="preconnect" href="${url}" crossorigin />`
+      preconnectTag += `<link rel="preconnect" href="${url}" crossorigin /> <link rel="dns-prefetch" href="${url}" crossorigin />`
```

Comment:
`preconnect` is supported since Chrome 46, Edge 79, Safari 11.1, and Opera 33.3, which is good enough. While IE 10-11 and Edge 12-18 are deprecated, we don't need to add extra `dns-prefetch` just for those browsers. As for Firefox, neither `preconnect` nor `dns-prefetch` is supported.

In short, adding extra `dns-prefetch` only results in bigger HTML, which is actually a negative optimization.

Reply:
Ok :)